Login integration version 2

sagarbadu opened the pull request:
Login integration version 2.

sagarbadu committed:
Login int

Login integration version 2

n-grounds commented on src/App.js:
```diff
@@ -1,34 +1,154 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import fire from "./services/firebase";
+import Login from"./Login"
```
functionally it won't matter, but for coding style consistency there should be a space after `from`

n-grounds commented on src/App.js:
```diff
@@ -1,34 +1,154 @@
- </BrowserRouter>
+    <div className = "App">
+      {user ? (
+        <Dashboard handleLogout={handleLogout} />
```
Seems odd that `user` isn't passed to the Dashboard component as well (like it is to the Login component).  Maybe it would be nice to pass it along and display a basic acknowledgement of who the user is, such as a "Hi, {user.name}" somewhere.

logan-giese committed:
Reorganization and formatting for merge

Dashboard.js has been restored to App.js, and App.js (login version) has been renamed to OpeningPage.js

logan-giese reviewed (approved):
After some fixes for file organization and formatting, it's finally ready to go.  Tested and it seems to work great.

logan-giese committed:
Merge branch 'dev' into login-integration-v2
